{{ cookiecutter.package_name|slugify }}/Dockerfile

```diff
@@ -19,7 +19,28 @@ ENV VIRTUAL_ENV /opt/app-env
 # Set the working directory.
 WORKDIR /app/
 
-FROM base as dev
+FROM base as deps
+
+# Install the run time Python environment.
+COPY poetry.lock* pyproject.toml /app/
+RUN --mount=type=cache,id=poetry,target=/root/.cache/ \
+    {%- if cookiecutter.private_package_repository_name %}
+    --mount=type=secret,id=poetry_auth,target=/root/.config/pypoetry/auth.toml \
+    {%- endif %}
+    mkdir -p src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/ && touch src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/__init__.py && touch README.md && \
+    poetry install --no-dev --no-interaction
+
+FROM deps as ci
+
+# Install the development Python environment.
+COPY .pre-commit-config.yaml /app/
```

Comment:
I don't think this line is necessary in the `ci` stage?

Reply:
I think we need it as we run `poe lint` using that image, which runs `pre-commit`.

Reply:
Yes, but shouldn't it use the PR branch's `.pre-commit-config.yaml`, not the one that happens to be in the image?

Reply:
Correct. Fixed